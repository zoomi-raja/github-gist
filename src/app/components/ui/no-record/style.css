.no-record{
  height: 50vh;
  width: 100%;
  display: flex;
  flex-direction: column;
  text-align: center;
  justify-content: center;
  background-color: rgba(0, 0, 0, 0.5);
}