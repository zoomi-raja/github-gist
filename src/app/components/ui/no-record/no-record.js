import './style.css';

export const NoRecord = () => {
	return <div className='no-record'><span>No Result Found</span></div>;
};