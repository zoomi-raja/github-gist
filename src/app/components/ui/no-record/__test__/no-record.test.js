import renderer from 'react-test-renderer';
import { NoRecord } from '../no-record';

it('no-record snapshot', () => {
  const tree = renderer.create(<NoRecord />).toJSON();
  expect(tree).toMatchSnapshot();
})