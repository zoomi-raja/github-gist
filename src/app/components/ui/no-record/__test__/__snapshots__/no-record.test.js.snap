// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-record snapshot 1`] = `
<div
  className="no-record"
>
  <span>
    No Result Found
  </span>
</div>
`;
